device/iot-device-client/src/main/java/com/microsoft/azure/sdk/iot/device/DeviceClient.java

```diff
@@ -376,6 +378,11 @@ private void commonConstructorVerifications() throws UnsupportedOperationExcepti
      */
     public void open() throws IOException
     {
+        if (this.ioTHubConnectionType == IoTHubConnectionType.USE_MULTIPLEXING_CLIENT)
+        {
+            throw new UnsupportedOperationException("Cannot open a multiplexed client through this method. Must use multiplexingClient.registerDeviceClient(deviceClient)");
```

Comment:
nit: I would change and construct this string by using the class and method names so we won't have any refactoring issues.

Reply:
Strings outside of javadocs don't support the {@ link } notation, if that is what you are getting at.

Reply:
ah, there is no equivalent of nameof in java. Fair enough